velox/exec/OutputBuffer.cpp

```diff
@@ -724,15 +724,15 @@ void OutputBuffer::getData(
 
     VELOX_CHECK_LT(destination, buffers_.size());
     auto* buffer = buffers_[destination].get();
-    VELOX_CHECK_NOT_NULL(
-        buffer,
-        "getData received after its buffer is deleted. Destination: {}, sequence: {}",
-        destination,
-        sequence);
-    freed = buffer->acknowledge(sequence, true);
-    updateAfterAcknowledgeLocked(freed, promises);
-    data = buffer->getData(
-        maxBytes, sequence, notify, activeCheck, arbitraryBuffer_.get());
+    if (buffer) {
```

Comment:
Let's update comments for this method in the header file to mention this behavior.

Reply:
Updated the comment for `deleteResults`

Reply:
Thanks. Would you also update comments for OutputBufferManager::getData API as this is the public API used by Prestissimo?